docs/supported_platforms.md

```diff
@@ -34,14 +34,13 @@ By these criteria, Shadow's oldest supported kernel version is currently 5.4
 ## Docker
 
 If you are installing Shadow within a Docker container, you must increase the
-size of the container's `/dev/shm` mount, add the "exec" mount option, and
-disable the seccomp security profile. You can do this by passing additional
-flags to `docker run`.
+size of the container's `/dev/shm` mount and disable the seccomp security
+profile. You can do this by passing additional flags to `docker run`.
 
 Example:
 
 ```bash
-docker run -it --tmpfs /dev/shm:rw,nosuid,nodev,exec,size=1024g --security-opt seccomp=unconfined ubuntu:22.04
+docker run -it --tmpfs /dev/shm:rw,nosuid,nodev,size=1024g --security-opt seccomp=unconfined ubuntu:22.04
```

Comment:
I think we can just use the simpler `--shm-size=1024g` instead of specifying all of the mount parameters.